.github/workflows/next-container-build.yaml

```
@@ -0,0 +1,87 @@
+name: Next CLI container build
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - main
+
```

we can ignore the `docs/` for this flow 
```suggestion
    paths-ignore: docs/
```